### PR TITLE
Update 'CLI11' to 1.7.1

### PR DESCRIPTION
**Pull request by henryiii**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/update.html)
  step by step carefully. **[Yes]**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://travis-ci.org/henryiii/hunter/builds/483841534

### CODE REVIEW

**Comment by ruslo** [comment]
> Here is the links to the Travis/AppVeyor with status "All passed":
>    https://travis-ci.org/henryiii/hunter/builds/483841534

That's not a package testing, it's documentation.

**Comment by ruslo** [comment]
Testing:
- https://ci.appveyor.com/project/ingenue/hunter/builds/21974169
- https://travis-ci.org/ingenue/hunter/builds/485989357?utm_source=github_status&utm_medium=notification

File: cmake/projects/CLI11/hunter.cmake [hunk 1]
```diff
@@ -54,6 +54,29 @@ hunter_add_version(
...
+
+hunter_add_version(
+    PACKAGE_NAME
+    CLI11
+    VERSION
+    1.7.0
+    URL
+    "https://github.com/CLIUtils/CLI11/archive/v1.7.0.tar.gz"
+    SHA1
+    5e1daa3ee03d17e040242d77c53e80747e03c220
```
Comment (ruslo) [inline cmake/projects/CLI11/hunter.cmake]: SHA1 is `223a3c3dff2ccbb37e9caafb3f6308fc32564060`

Comment (henryiii) [reply cmake/projects/CLI11/hunter.cmake]: For some reason curl followed by sshsum was giving the wrong values on my Mac. Reran the downloads and checks using wget on Linux and that worked.

**Comment by henryiii** [comment]
> That's not a package testing, it's documentation.

That seems really familiar, I think I made the same mistake last time I updated the package.

**Comment by ruslo** [comment]
Testing:
- https://ci.appveyor.com/project/ingenue/hunter/builds/22019198
- https://travis-ci.org/ingenue/hunter/builds/486807144?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
- https://ci.appveyor.com/project/ingenue/hunter/builds/22021334
- https://travis-ci.org/ingenue/hunter/builds/486846390?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.94

**Pull request merged**